### ISSUE TITLE
CTRL+E keyboard shortcut to run the save as (hence export) dialog straight away

### ISSUE THREAD

**Issue by marek-lach** [vkbo/novelWriter#1394]
`CTRL+E` as a quick `Save As` dialog shortcut could be handy, if the entire written manuscript as is needs to be exported quickly into a different format.

**Comment by vkbo** [comment]
That's a pretty good idea. I can add a Quick Build option to the new tool. However, there will need to be a setting for this as there will be multiple build definitions in the new tool.

The shortcut `Ctrl+E` cannot be used as it is planned for something else, but I'm sure a suitable one can be found.

**Comment by marek-lach** [comment]
> The shortcut `Ctrl+E` cannot be used as it is planned for something else, but I'm sure a suitable one can be found.

There’s the option of `CTRL+P` perhaps, as in print (even when only a virtual export, as in printing to a file essentially)…

**Comment by vkbo** [comment]
No, not `Ctrl+P` either. I want to use that for feature #1001 I think. I'm considering something with a function key since the build tool sits on `F5`.
